lib/widgets/app.dart

```diff
@@ -161,6 +165,40 @@ class _ZulipAppState extends State<ZulipApp> with WidgetsBindingObserver {
     super.initState();
     WidgetsBinding.instance.addObserver(this);
     UpgradeWelcomeDialog.maybeShow();
+
+    // Move to a AndroidShareService or similar.
+    ZulipBinding.instance.androidIntentEvents.forEach((androidIntentEvent) async {
+      assert(debugLog('androidIntentEvent.action: ${androidIntentEvent.action}'));
```

Comment:
Let's also make this callback a named function. Can be a private static method on the relevant class.